app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -28,6 +29,10 @@ def host_subscriptions_names(host)
         host.subscriptions.map(&:name)
       end
 
+      def host_redhat_subscriptions(host)
```

Comment:
It all works well. To be consistent with the other methods I think this one should end in _names such as `host_redhat_subscription_names`

Reply:
@jturel thank you, modified as suggested.